NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/JavaScriptSnippet.cs

```diff
@@ -34,6 +31,9 @@ public class JavaScriptSnippet : IJavaScriptSnippet
 
         private readonly JavaScriptEncoder encoder;
 
+        /// <summary>JavaScript snippet.</summary>
+        private string snippet = Resources.JavaScriptSnippet;
```

Comment:
I see why you changed this from `static`, so you can set the result of the string.replace on this same variable.
But I don't think this is necessary.

I believe a new instance of `JavaScriptSnippet` is going to be created for every request, so we would be unable to store the result of the string replace.
Plus you're only storing the first half (ikey) and not the second half (auth context).

I think you can change this back to the static variable `static readonly string Snippet`.
In the `ScriptBody()` method, you can store the result of string Replace in a local variable. `var snippet = this.Snippet.Replace`

Reply:
restored static readonly snippet